python/popgetter/metadata.py

```diff
@@ -10,6 +11,64 @@
 from pydantic import BaseModel, Field, computed_field, model_validator
 
 
+class COL(Enum):
+    """
+    This class stores the eventual column names of all the metadata classes,
+    which are used when serialising the metadata to a dataframe.
+    """
+
+    GEO_ID = "GEO_ID"
+
+    COUNTRY_ID = "country_id"
+    COUNTRY_NAME_SHORT_EN = "country_name_short_en"
+    COUNTRY_NAME_OFFICIAL = "country_name_official"
+    COUNTRY_ISO3 = "country_iso3"
+    COUNTRY_ISO2 = "country_iso2"
+    COUNTRY_ISO3166_2 = "country_iso3166_2"
+
+    PUBLISHER_ID = "data_publisher_id"
+    PUBLISHER_NAME = "data_publisher_name"
+    PUBLISHER_URL = "data_publisher_url"
+    PUBLISHER_DESCRIPTION = "data_publisher_description"
+    PUBLISHER_COUNTRIES_OF_INTEREST = "data_publisher_countries_of_interest"
+
+    GEOMETRY_ID = "geometry_id"
+    GEOMETRY_FILENAME_STEM = "geometry_filename_stem"
+    GEOMETRY_VALIDITY_PERIOD_START = "geometry_validity_period_start"
+    GEOMETRY_VALIDITY_PERIOD_END = "geometry_validity_period_end"
+    GEOMETRY_LEVEL = "geometry_level"
+    GEOMETRY_HXL_TAG = "geometry_hxl_tag"
+
+    SOURCE_ID = "source_id"
+    SOURCE_NAME = "source_name"
+    SOURCE_DATE_PUBLISHED = "source_date_published"
+    SOURCE_REFERENCE_PERIOD_START = "source_reference_period_start"
+    SOURCE_REFERENCE_PERIOD_END = "source_reference_period_end"
+    SOURCE_COLLECTION_PERIOD_START = "source_collection_period_start"
+    SOURCE_COLLECTION_PERIOD_END = "source_collection_period_end"
+    SOURCE_EXPECT_NEXT_UPDATE = "source_expect_next_update"
+    SOURCE_URL = "source_url"
+    SOURCE_DATA_PUBLISHER_ID = "source_data_publisher_id"
+    SOURCE_DESCRIPTION = "source_description"
+    SOURCE_GEOMETRY_METADATA_ID = "source_geometry_metadata_id"
+
+    METRIC_ID = "metric_id"
+    METRIC_HUMAN_READABLE_NAME = "metric_human_readable_name"
+    METRIC_SOURCE_METRIC_ID = "metric_source_id"
+    METRIC_DESCRIPTION = "metric_description"
+    METRIC_HXL_TAG = "metric_hxl_tag"
+    METRIC_PARQUET_PATH = "metric_parquet_path"
+    METRIC_PARQUET_COLUMN_NAME = "metric_parquet_column_name"
+    METRIC_PARQUET_MARGIN_OF_ERROR_COLUMN = "metric_parquet_margin_of_error_column"
+    METRIC_PARQUET_MARGIN_OF_ERROR_FILE = "metric_parquet_margin_of_error_file"
+    METRIC_POTENTIAL_DENOMINATOR_IDS = "metric_potential_denominator_ids"
+    METRIC_PARENT_METRIC_ID = "metric_parent_id"
```

Comment:
Perhaps we should also name this one with the same pattern?
```suggestion
    METRIC_PARENT_METRIC_ID = "metric_parent_metric_id"
```

Reply:
Oops. I'd blame Copilot but knowing me, Copilot probably gave me the right thing and I deleted it for some reason.